lib/validate-session-invitation.js

```diff
@@ -32,6 +33,16 @@ function validateSessionInvitation (args, callback) {
     });
   }
 
+  function checkCapacity (ticket, wfCb) {
+    seneca.act({ role: plugin, cmd: 'searchApplications', query: { ticketId: ticket.id } }, (err, applications) => {
```

Comment:
I think this should only check approved tickets. So it would succeed if say there was a capacity of 10, 20 applications, but only 5 approved applications. Thoughts @conoramurphy?